docs/implementation/spec-to-implementation.md

```diff
@@ -7,6 +7,8 @@ _Work in Progress_
 
 ### BYO Software
 
+The K8TRE Reference Implementation will be agnostic to whether a deployment supports "bring-your-own software and code" versus curated software and will be offered with both options available via deployment configuration.
```

Comment:
I wonder if we should say the reference implementation includes some example open-source applications but we expect deployments to use their own container images? I think it's unlikely that anyone would use the default set of applications.